### ISSUE TITLE
Multiple Maps along the x and y axis do not Render correctly

### ISSUE THREAD

**Issue by NoahShomette** [Droggelbecher/bevy-fast-tilemap#7]
I'm attempting to use this crate alongside a chunking implementation. To do so I wanted to make it so that each Chunk was a separate fast tilemap. However doing so causes all the other maps except for the first map to not be rendered correctly. 

I copied the code straight from the examples for spawning my maps.

You can see based on the image that all the maps should be random like the first one however none of them actually are.

![bevy_fast_tilemap_example_EP54SEQPqE](https://github.com/Droggelbecher/bevy-fast-tilemap/assets/106724696/84943331-4d99-42e6-b95d-9b0af20ed92d)

Code for spawning the tilemaps, used for each tilemap

```rust
// Create map with the given dimensions of our chunk
        let map = Map::builder(
            // Map size (tiles)
            uvec2(
                chunk.get_chunk_dimensions().x,
                chunk.get_chunk_dimensions().y,
            ),
            // Tile atlas
            asset_server.load("tiles_16.png"),
            // Tile size (pixels)
            vec2(16., 16.),
        )
        .build_and_set(&mut images, |pos| rng.gen_range(0..15));

        commands
            .entity(entity)
            .insert((
                SpatialBundle {
                    transform: Transform {
                        translation: Vec3::new(
                            chunk.chunk_pos.x() as f32
                                * chunk.get_chunk_dimensions().x as f32
                                * 16.0,
                            chunk.chunk_pos.y() as f32
                                * chunk.get_chunk_dimensions().y as f32
                                * 16.0,
                            1.0,
                        ),
                        ..default()
                    },
                    ..default()
                },
                ChunkMapSpawned,
            ))
            .with_children(|parent| {
                let mut map_bundle = MapBundle::new(map);
                map_bundle.transform.translation = Vec3::new(
                    chunk.chunk_pos.x() as f32 * chunk.get_chunk_dimensions().x as f32 * 16.0,
                    chunk.chunk_pos.y() as f32 * chunk.get_chunk_dimensions().y as f32 * 16.0,
                    1.0,
                );
                parent
                    .spawn(map_bundle)
                    .insert(Transform::from_translation(Vec3::new(1.0, 1.0, 1.0)))
                    // Have the map manage our mesh so it always has the right size
                    .insert(MeshManagedByMap)
                    .insert(FastTileMap);
            });

```

**Comment by Droggelbecher** [comment]
Hey @NoahShomette , thanks for the report. Seems this is indeed a bug as so far the tilemap shader completely ignores the entities `GlobalTransform`. I'll try and address it this week (if I don't manage to, be warned itll take much longer because of upcoming vacation)

**Comment by Droggelbecher** [comment]
@NoahShomette Could you check whether https://github.com/Droggelbecher/bevy-fast-tilemap/pull/8 fixes your issue? I also included a minimalistic "chunked" example. It should now correctly support translation (but no rotation/scaling).

**Comment by NoahShomette** [comment]
@Droggelbecher Yep that fixed it and they now render correctly across multiple chunks for me! Thank you very much!

**Comment by Droggelbecher** [comment]
Solved by #8

**Comment by Droggelbecher** [comment]
Released as v0.5.1
